<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1335</width>
    <height>938</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>1271</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>Previous Lines:</string>
   </property>
   <widget class="QTableView" name="lines_view">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>1271</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>1271</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Info</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>193</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Document:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="document">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>168</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Date:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="date">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>369</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Partner:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="partner">
       <property name="minimumSize">
        <size>
         <width>321</width>
         <height>0</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>40</y>
      <width>200</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Warehouse:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="warehouse">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>611</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>Available Physical Stock:</string>
   </property>
   <widget class="QTableView" name="stock_view">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>611</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>480</y>
     <width>631</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>Selected Stock Per Line:</string>
   </property>
   <widget class="QTableView" name="defined_view">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>631</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="delete_">
   <property name="geometry">
    <rect>
     <x>1217</x>
     <y>800</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>800</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>880</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>880</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>942</x>
     <y>900</y>
    </hint>
    <hint type="destinationlabel">
     <x>796</x>
     <y>851</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
